frontend_tests/puppeteer_tests/message-basics.ts

```diff
@@ -259,21 +259,6 @@ async function search_tests(page: Page): Promise<void> {
     );
 }
 
-async function expect_all_pm(page: Page): Promise<void> {
-    await page.waitForSelector("#zfilt", {visible: true});
-    await common.check_messages_sent(page, "zfilt", [
-        ["You and Cordelia, Lear's daughter, King Hamlet", ["group pm a", "group pm b"]],
-        ["You and Cordelia, Lear's daughter", ["pm c"]],
-        ["You and Cordelia, Lear's daughter, King Hamlet", ["group pm d"]],
-        ["You and Cordelia, Lear's daughter", ["pm e"]],
-    ]);
-    assert.strictEqual(
-        await common.get_text_from_selector(page, "#left_bar_compose_stream_button_big"),
-        "New stream message",
-    );
-    assert.strictEqual(await page.title(), "Private messages - Zulip Dev - Zulip");
-}
```

Comment:
I don't think we should need to remove this test.

Reply:
yess, since now we've a  `ALL PRIVATE MESSAGES` icon, clicking on which takes us to `#narrow/is/private` so we can test this in `puppeteer`, 

I've added this test cases which now clicks on the icon of `all_private_messages` and expects to load All Private messages.

Thanks :))